docker_templates/templates/docker_images/create_ros2_ros1_bridge_image.Dockerfile.em

```diff
@@ -0,0 +1,80 @@
+@(TEMPLATE(
+    'snippet/add_generated_comment.Dockerfile.em',
+    user_name=user_name,
+    tag_name=tag_name,
+    source_template_name=template_name,
+))@
+@(TEMPLATE(
+    'snippet/from_base_image.Dockerfile.em',
+    template_packages=template_packages,
+    os_name=os_name,
+    os_code_name=os_code_name,
+    arch=arch,
+    base_image=base_image,
+    maintainer_name=maintainer_name,
+))@
+@{
+template_dependencies = []
+# add 'python3-pip' to 'template_dependencies' if pip dependencies are declared
+if 'pip3_install' in locals():
+    if isinstance(pip3_install, list) and pip3_install != []:
+        template_dependencies.append('python3-pip')
+}@
+@(TEMPLATE(
+    'snippet/install_upstream_package_list.Dockerfile.em',
+    packages=template_dependencies,
+    upstream_packages=upstream_packages if 'upstream_packages' in locals() else [],
+))@
+@
+@[if 'pip3_install' in locals()]@
+@[  if pip3_install]@
+# install python packages
+RUN pip3 install -U \
+    @(' \\\n    '.join(pip3_install))@
+
+@[  end if]@
+@[end if]@
+
+# setup keys
+RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 421C365BD9FF1F717815A3895523BAEEB01FA116
+
+# setup sources.list
+RUN echo "deb http://packages.ros.org/ros/ubuntu `lsb_release -sc` main" > /etc/apt/sources.list.d/ros-latest.list
+
+ENV ROS1_DISTRO @rosdistro_name
```

Comment:
An this to be:
ENV ROS1_DISTRO @ros1distro_name

and then update these accordingly:  
https://github.com/osrf/docker_images/blob/b5dbb565f7ebd92dfd1d85aa8c0be6e657979b7b/ros2/manifest.yaml#L23-L46  
https://github.com/osrf/docker_images/blob/b5dbb565f7ebd92dfd1d85aa8c0be6e657979b7b/ros2/.config/platform.yaml.em#L7-L8